diktat-common/src/main/kotlin/org/cqfn/diktat/common/config/rules/RulesConfigReader.kt

```diff
@@ -85,6 +84,18 @@ open class RulesConfigReader(override val classLoader: ClassLoader) : JsonResour
     }
 }
 
+fun List<RulesConfig>.getCommonConfiguration() = lazy { CommonConfiguration(getCommonConfig()) }
+
+class CommonConfiguration(commonConfig: RulesConfig?) {
```

Comment:
Why not inherit from `RuleConfiguration` too? Also, I think this class can be constructed just from a configuration map, no need to keep a whole `RulesConfig` instance here.